toml_edit fails to compile due to IndexMap

I can't seem to compile version `0.19.4` due to a missing argument when using the IndexMap create. 

```
~/g/x (main)> cargo build
...
   Compiling toml_edit v0.19.4
error[E0107]: this struct takes 3 generic arguments but 2 generic arguments were supplied
   --> /home/y/.cargo/registry/src/github.com-1ecc6299db9ec823/toml_edit-0.19.4/src/table.rs:457:33
    |
457 | pub(crate) type KeyValuePairs = IndexMap<InternalString, TableKeyValue>;
    |                                 ^^^^^^^^ --------------  ------------- supplied 2 generic arguments
    |                                 |
    |                                 expected 3 generic arguments
    |
note: struct defined here, with 3 generic parameters: `K`, `V`, `S`
   --> /home/y/.cargo/registry/src/github.com-1ecc6299db9ec823/indexmap-1.9.2/src/map.rs:76:12
    |
76  | pub struct IndexMap<K, V, S> {
    |            ^^^^^^^^ -  -  -
help: add missing generic argument
    |
457 | pub(crate) type KeyValuePairs = IndexMap<InternalString, TableKeyValue, S>;
    |                                                                       +++

For more information about this error, try `rustc --explain E0107`.
   Compiling gdk4-sys v0.5.5
error: could not compile `toml_edit` due to previous error
warning: build failed, waiting for other jobs to finish...
```

--- a/crates/toml_edit/Cargo.toml
+++ b/crates/toml_edit/Cargo.toml
@@ -47,7 +47,7 @@ serde = ["dep:serde", "toml_datetime/serde", "dep:serde_spanned"]
 unbounded = []
 
 [dependencies]
-indexmap = "1.9.1"
+indexmap = { version = "1.9.1", features = ["std"] }
 winnow = "0.3.0"
 serde = { version = "1.0.145", optional = true }
 kstring = { version = "2.0.0", features = ["max_inline"], optional = true }

